ios/PushNotificationPlugin/PushNotificationPlugin.m

```diff
@@ -5,14 +5,19 @@
 #import "UALocationService.h"
 #import "UAConfig.h"
 #import "NSJSONSerialization+UAAdditions.h"
+#import "SECommandQueue.h"
```

Comment:
This is something defined in the shoutem project? Importing this here means that this plugin cannot be used outside of the shoutem app. Can we extract this queue into some external library so the plugin remains independent of the shoutem app?